halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/azure/AzureCommandProperties.java

```diff
@@ -42,4 +42,6 @@
   static final String IMAGE_SKU_DESCRIPTION = "The SKU for your base image. See https://aka.ms/azspinimage to get a list of images.";
 
   static final String IMAGE_VERSION_DESCRIPTION = "The version of your base image. This defaults to 'latest' if not specified.";
+
+  static final String REGIONS_DESCRIPTION = "The Azure regions this Spinnaker account will manage. If not specified, the default value is 'eastus,westus'";
```

Comment:
I realize I asked you to add the default text here. Sorry about that. I think it makes more sense to just rely on the doc generator to pick up the actual field defaults. It will also avoid the situation where the defaults for the edit command are (incorrectly) described as being `eastus,westus`.

Reply:
Thanks for explanation, removed the default value description here